queenbee/cli/config/auth.py

```diff
@@ -0,0 +1,48 @@
+from ...config.auth import PollinationAuth
+
+try:
+    import click
+    from click_plugins import with_plugins
+except ImportError:
+    raise ImportError(
+        'click modules not installed. Try `pip install queenbee[cli]` command.'
+    )
+
+@click.group('auth')
+def auth():
+    """manage queenbee configuration auth
+    
+
+    """
+    pass
+
+@auth.group('add')
+def add():
+    """add auth domains and methods to your queenbee config
+    
+
+    """
+    pass
+
+@add.command('pollination')
```

Comment:
I believe this should be added from queenbee-pollination not in pollination itself. Otherwise we will have to add several plugin commands to the core library.

Reply:
This is just a command to add "pollination" style Auth to the queenbee config file. It will not serve as an entry point for other pollination commands.

Reply:
I understand that this is not the entry point command but don't you agree that this is a plugin specific command?

What stops you to move this command to queenbee-pollination? You can import `add` and then add your new command when queenbee-pollination is loaded. Right?

Reply:
Yeah it does kinda feel like a plugin command. The difficulty I'm having is not with integrating the click/cli side as a plugin but managing passing the `config` object which contains an Pollination api token and can create a JWT token that is then used to authenticate certain requests within recipes.

Essentially if I make this command a plugin then it feels I have to make this [class](https://github.com/ladybug-tools/queenbee/pull/97/commits/84add41f8789f2188eae41a36d8b72f747456f01#diff-22a2a60dc5ac2d09ddd52a6343eae14cR35-R93) which then makes it difficult (I think) to handle authentication to pollination servers when managing recipe dependencies. Part of the reason this is difficult is because we don't (currently) implement standard authentication mechanisms which we can expect to be reusable by other non-pollination servers.